R/jointScoreTest.R

```diff
@@ -20,14 +20,14 @@ jointScoreTest <- function(nullmod, G) {  # # Check rownames/colnames match.
   }
 
   # Reorder samples.
-  idx <- match(rownames(nullmod$model.matrix), rownames(G))
+  idx <- match(rownames(null.model$model.matrix), rownames(G))
   G <- G[idx, , drop = FALSE]
 
   # Genotype adjusted for covariates and random effects.
-  Gtilde <- calcGtilde(nullmod, G)
+  Gtilde <- calcGtilde(null.model, G)
```

Comment:
@smgogarten This will fail with a "small" null model. Should there be any sort of check or warning?

Reply:
If we want to check if null models are "small", then maybe the null model should have a list element indicating if it's "small' or "full". This would be pretty quick and could be added when making other null model updates.

An alternative solution is turn null models into S4 Classes (a base class, and then extend that for small vs full null models, maybe). Then you could specify in the various signatures what you needed. I think that's the "best practices" way, but it would be a big change.